app/code/Magento/Bundle/Test/Mftf/Test/CurrencyChangingBundleProductInCartTest.xml

```diff
@@ -40,7 +40,7 @@
                 <click selector="{{CurrencySetupSection.currencyOptions}}" stepKey="closeOptions"/>
                 <waitForPageLoad stepKey="waitForCloseOptions"/>
                 <click stepKey="saveUnselectedConfigs" selector="{{AdminConfigSection.saveButton}}"/>
-                <amOnPage url="{{AdminLogoutPage.url}}" stepKey="logout"/>
+                <actionGroup ref="AdminLogoutActionGroup" stepKey="amOnLogoutPage"/>
```

Comment:
If you can keep the original `stepKey` that would be appreciated, as some developers might have attached their Actions to it.

Reply:
**Done**